policyengine_us/parameters/gov/states/hi/tax/income/subtractions/military_pay/max_amount.yaml

```diff
@@ -0,0 +1,19 @@
+description: Hawaii caps the military reserve or Hawaii national guard duty pay exclusion at this amount.
+values: 
+  2021-01-01: 7_345
+
+metadata:
+  label: Hawaii military reserve or national guard duty pay exclusion max amount
+  period: year
+  unit: currency-USD
+  reference:
+  - title: Hawaii Resident Income Tax Instructions N-11 Rev.2022 
+    href: https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=13
+  - title: Hawaii 2022 Individual Income Tax Return N-11 From Rev.2022
+    href: https://files.hawaii.gov/tax/forms/2022/n11_i.pdf#page=2
+  - title: Hawaii Resident Income Tax Instructions N-11 Rev.2021 
+    href: https://files.hawaii.gov/tax/forms/2021/n11ins.pdf#page=13
+  - title: Hawaii 2021 Individual Income Tax Return N-11 From Rev.2021
+    href: https://files.hawaii.gov/tax/forms/2021/n11_i.pdf#page=2  
+  - title: Hawaii income tax low §235-100 Persons in military service
```

Comment:
I dont think I see the cap in this source

Reply:
I found the Exemption for National Guard, military reserves, and other active duty military personnel claiming Hawaii as their residence of record here:  https://law.justia.com/codes/hawaii/2022/title-14/chapter-249/section-249-6-5/. But it doesn't contain the value for the cap. Will this work?

Reply:
whenever we add references for parameter values, they always have to include the actual value (or a previously unadjusted amount), if it is not mentioned in the law we have to add a comment stating "# The cap value is mentioned in the legal code"